src/style.css

```diff
@@ -1 +1,73 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't reset margins and paddings using  * selector.
With many items on the page it may be performance-heavy.
Use tags for that,

Border-box, however, is fine to use with *